docs/userguide/services.md

```diff
@@ -130,26 +132,12 @@ numbers, formatting dates and times, and formatting messages:
 
 #### General Formatting
 
-*   [Format](index.md)
-    Format is the abstract superclass of all format classes. It provides the
-    basic methods for formatting and parsing numbers, dates, strings, and other
-    objects.
-
-*   [FieldPosition](index.md) (§)
-    FieldPosition is a concrete class for holding the field constant and the
-    beginning and ending indices for the number and date fields.
-
-*   [ParsePosition](index.md) (§)
-    ParsePosition is a concrete class for holding the parse position in a string
-    during parsing.
+See [Formatting and Parsing Classes](formatparse/index.md#Formatting_and_Parsing_Classes) for an introduction to the following:
```

Comment:
ditto, please check all of these anchors/fragments

Reply:
done